src/corefx/System.Globalization.Native/pal_calendarData.c

```diff
@@ -385,20 +357,32 @@ bool EnumSymbols(const char* locale,
     for (int32_t i = startIndex; i < symbolCount; i++)
     {
         UErrorCode ignore = U_ZERO_ERROR;
-        int symbolLen = udat_getSymbols(pFormat, type, i, nullptr, 0, &ignore);
+        int symbolLen = udat_getSymbols(pFormat, type, i, NULL, 0, &ignore) + 1;
 
-        std::vector<UChar> symbolBuf(symbolLen + 1, '\0');
+        UChar* symbolBuf = calloc(symbolLen, sizeof(UChar*));
```

Comment:
The same question as in a similar case before - why are you allocating one less UChar than the original code?

Reply:
I added 1 to `symbolLen` above